### ISSUE TITLE
Como Configurar sua Wallet com MEMO no Hive.OS?

### ISSUE THREAD

**Issue by Wallis-Soares** [RainbowMiner/RainbowMiner#2353]
Sou novo no campo da mineração e ao tentar criar um flight Sheet para a moeda Iron, me deparei com a questão do MEMO, porém no Hive.os só tem campo para a carteira. Alguém pode me ajudar?

**Comment by RainbowMiner** [comment]
Yes, just add the memo to the wallet address by using a `+` (e.g. 0102030405060708+0a0b0c0d0e)

**Comment by Wallis-Soares** [comment]
Thank's a lot. What aborto this: "You need to use Iron Fish node v1.0.0 (or latest) to see your payments." Tô recieve my money need do I something different? I created my wallet with memo, created The fligthsheet but a don,t recieve nothing yet.

**Comment by RainbowMiner** [comment]
Did you mine to a pool or did you solo-mine?

**Comment by whitindllc** [comment]
okay this is for ironfish? how do i set this up... i have the node running and a wallet...

do i just go to the gui and turn on herominers and then go to coins and add a coin or do i need to go to the pool and add the coin there... this has always confused me....

**Comment by RainbowMiner** [comment]
Yes, you will have to add the coin to the pool to enable mining for that currency.
The coins setup is a global place to store your wallets, but they have still to be assigned to the pools. Use `$` in front of the coin symbol to refer to the coins database (it is like a variable)
![Screenshot_20230426_095719_Brave](https://user-images.githubusercontent.com/39437538/234509002-057c29a6-91b7-4365-9344-934fb8128a60.jpg)

**Comment by Wallis-Soares** [comment]
Estou minerando pela Herominers e está dando certo.  Não vi dificuldade nenhuma. Estou recebendo as moedas na minha wallet normalmente.  Sugiro Kucoin onde o preço é melhor que a SafeTrade e menos complicada.

**Comment by RainbowMiner** [comment]
> I'm mining for Herominers and it's working. I didn't see any difficulty. I am receiving coins in my wallet normally. I suggest Kucoin where the price is better than SafeTrade and less complicated.

Yes, in fact Kucoin (highest trading volume) and Gate.io (second highest trading volume) are good options to mine Ironfish via Herominers. SafeTrade has very low volume, so it's harder to sell the coins there.

**Comment by whitindllc** [comment]
> Yes, you will have to add the coin to the pool to enable mining for that currency. The coins setup is a global place to store your wallets, but they have still to be assigned to the pools. Use `$` in front of the coin symbol to refer to the coins database (it is like a variable) ![Screenshot_20230426_100117_Brave](https://user-images.githubusercontent.com/39437538/234510055-f0446ad0-858c-495e-88b9-c780c4a68634.jpg)

okay i didnt do this... i just found the pool and added a coin to it and then put the address in.... herominers... then added IRON and the address.. then added a check mark in the pools for it....

this other route with adding IRON and then $IRON does what? vs what i did

**Comment by Wallis-Soares** [comment]
Alguém percebeu um aumento na temperatura das RIGs? As minhas subiram uns 15°C.

**Comment by Wallis-Soares** [comment]
Minhas AMD RX 580 caíram pela metade de produtividade na mineração de IRON. Alguém sabe o motivo? Dois dias atrás estavam fazendo 26GH e agora estão fazendo 13GH cada.
